### ISSUE TITLE
[0.50.0-SNAPSHOT] RFE - Add a way to have Princess ignore units by player/force

### ISSUE THREAD

**Issue by PhoenixHeart512** [MegaMek/mekhq#4568]
### Description
I would like to have a way to tell Princess to ignore all units of a specific player or force. The example that comes to mind is I would like to be able to tell Princess to ignore all Convoy units and only engage the enemy escorts. As an example of the reverse, sometimes I would like Princess to focus her attention on destroying evacuating units while I deal with the escorts.

**Comment by PhoenixHeart512** [comment]
For extra credit, I wouldn't mind also having a way to set ignore targets via the GUI, but even just a text command such as "ignore: player1" would be great.

**Comment by PhoenixHeart512** [comment]
I think I submitted this in the wrong project, this seems like it should have been in the Megamek project since it involves ignoring targets in-game?

**Comment by OrbMonky** [comment]
Adding support for this, Convoy lances tend to run past their escorts making them the 'only' target Princess sees, and/or have lower TN's than real fighting forces. If your allies field artillery/aero/ground forces they tend to just obliterate them uselessly, costing support points and contributing little to the fight unless you micromanage them which isn't really the point of Princess.

**Comment by Sleet01** [comment]
Yeah, this would be implemented in MM but then we'd pass info into the MM instance from MHQ.  Moving...
